gradle.properties

```diff
@@ -255,7 +255,7 @@ commonsCollectionsVersion=4.4
 # Pac4j versions
 ###############################
 pac4jSpringWebmvcVersion=6.0.3
-pac4jVersion=5.4.6
+pac4jVersion=5.7.2-SNAPSHOT
```

Comment:
This cannot happen. I realize you likely need this for `val sessionStore = new JEESessionStore();`. Instead, define a subclass for JEESessionStore and use that instead. Keep the pac4j version as is.

Reply:
Yes that is correct. Doing this violates the release policy, and possibly introduces (may) breaking changes for others.

Reply:
It perfectly makes sense. I shouldn't have forgotten